input/docs/getting-started/running-cake-scripts.md

```diff
@@ -0,0 +1,61 @@
+Order: 10
+---
+
+There are different runners available for running Cake scripts.
+
+# .NET Core Tool
+
+:::{.alert .alert-info}
+This is the recommended way to run Cake Scripts.
+:::
+
+The [Cake.Tool](https://www.nuget.org/packages/Cake.Tool) NuGet package, is a .NET Core global tool compiled for .NET Core 2.1 or newer.
+
+:::{.alert .alert-info}
+The following instructions require .NET Core 3.0 or newer.
+See [How to manage .NET Core tools](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools) for details and other options.
+:::
+
+Make sure to have a tool manifest available in your repository or create one using the following command:
+
+```shell
+dotnet new tool-manifest
+```
+
+Install Cake as a local tool using the `dotnet tool` command:
+
+```shell
+dotnet tool install Cake.Tool --version x.y.z
+```
```

Comment:
Is it worth mentioning here how you run the "bootstrapping" on a new machine?  i.e. what steps need to be completed on a fresh clone of a repository to get the build to work, using this approach?

Reply:
Bootstrapping is something which I prefer to be handled on a separate page. See #931 for bootstrapping for .NET Framework runner. This page can be enhanced later to include bootstrapping information for .NET Core Tool

Reply:
Fair.  Once that is in place, I think we should link through to that from here.

Reply:
Just to expand on this a little bit, I was referring specifically to this: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-tool-restore where once the manifest file was created and added to source control, you would need to run `dotnet tool restore` and then `dotnet cake`

Reply:
I moved bootstrapping process for .NET Core tool to its own page. Bootstrapping process for .NET Framework runner will follow in a separate PR with #931.